pkgs/development/compilers/gcc/common/pre-configure.nix

```
@@ -27,6 +31,8 @@ in lib.optionalString (hostPlatform.isSunOS && hostPlatform.is64bit) ''
   export lib=$out;
 '' + lib.optionalString langAda ''
   export PATH=${gnat-bootstrap}/bin:$PATH
+'' + lib.optionalString (langD && lib.versionAtLeast version "12") ''
+  export PATH=${gdc}/bin:$PATH
```

```suggestion
'' + lib.optionalString (langD && lib.versionAtLeast version "12") ''
  export PATH=${gdc}/bin:$PATH
```

Can't this be accomplished by putting `gdc` in `nativeBuildInputs` instead?

(I suppose `langAda` ought to be doing this too)